commands/guild/twitch-announcer.js

```diff
@@ -147,227 +128,260 @@ module.exports = class TwitchAnnouncerCommand extends Command {
         .setTimestamp(DBInfo.date);
     }
 
-    //Check embed trigger
+    // Check Twitch Announcer Status
     if (textFiltered == 'check') {
-      if (currentMsgStatus == 'disable') message.channel.send(disabledEmbed);
-      else {
+      if (message.guild.twitchData.isRunning) {
         message.channel.send(enabledEmbed);
-        return;
+      } else {
+        message.channel.send(disabledEmbed);
       }
       return;
     }
-    //Disable Set
-    if (textFiltered == 'disable') {
-      currentMsgStatus = 'disable';
-      message.channel.send(disabledEmbed);
-    }
 
-    //Enable Set
+    // Enable Twitch Announcer
     if (textFiltered == 'enable') {
-      currentMsgStatus = 'enable';
+      if (message.guild.twitchData.isRunning == false) {
+        var failedAttempts = 0;
+        message.guild.twitchData.isRunning = true;
+        message.guild.twitchData.Interval = setInterval(async function() {
+          await announcer();
+        }, DBInfo.timer * 60000); //setInterval() is in MS and needs to be converted to minutes
+      }
       message.channel.send(enabledEmbed);
+      return;
+    }
 
-      //Ticker Section (Loop)
-      var Ticker = setInterval(async function() {
-        if (currentMsgStatus == 'disable') {
-          clearInterval(Ticker);
-          return;
-        }
+    // Disable Twitch Announcer
+    if (textFiltered == 'disable') {
+      message.guild.twitchData.isRunning = false;
+      message.guild.twitchData.Interval = clearInterval(
+        message.guild.twitchData.Interval
+      );
+      message.channel.send(disabledEmbed);
+      return;
+    }
 
-        let announcedChannel = message.guild.channels.cache.find(
-          channel => channel.id == DBInfo.channelID
+    async function announcer() {
```

Comment:
It's best to avoid defining a function if there's only one invocation of it. 
In line 146 you're defining an anonymous callback function and it all it does it call 'announcer'. You can basically copy the code of 'announcer' into the anonymous callback function block

Reply:
Roger that

Thank you for the explanation 

on it now




Much love
-Bacon